app/models/application_record.rb

```diff
@@ -9,10 +9,11 @@ class ApplicationRecord < ActiveRecord::Base
     name_desc = @meta[:name_desc] || "Name of the #{@meta[:friendly_name] || @meta[:class_scope]}#{meta_example}"
     property :name, String, desc: name_desc
     property :present?, one_of: [true, false], desc: 'Object presence (always true)'
+    property :previous_revision, Object, desc: "Previous revision of the record as tracked in the audit, returns the same object if there is no previous revision"
   end
 
-  class Jail < Safemode::Jail
-    allow :id, :name, :present?
+  class Jail < ::Safemode::Jail
+    allow :id, :name, :present?, :previous_revision
```

Comment:
would this work for resources that aren't audited?

Reply:
What would work? SafeMode will either work or throw a MethodNotFound exception as expected.

For Webhooks, all exposed models do have audit, we expose just couple of them at the moment.